students/10th/soojungsohn/user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import SignUpView, SignInView
```

Comment:
- `django`  자체 모듈과 우리가 생성한 파일을 import 하는 라인 사이에 한 줄 간격을 띄워주는 것이 위코드 코딩 컨벤션 입니다 :)
- 또한, 하나의 파일에서 여러 클래스를 Import 할 경우, 각 클래스마다 행을 분리해줍니다.

```suggestion
from django.urls import path

from .views import (
    SignUpView, 
    SignInView
)
```